daml-script/export/src/test/scala/com/daml/script/export/EncodeValueSpec.scala

```diff
@@ -43,12 +43,14 @@ class EncodeValueSpec extends AnyFreeSpec with Matchers {
           ),
         )
       )
-      encodeValue(Map.empty, Map.empty, r).render(80) shouldBe
-        """M.R1 with
```

Comment:
I don’t actually understand what broke. This looks like indentation of nested records works fine so what didn’t work? Can we capture that in a test?

Reply:
I added a test for list of record.